packages/router/src/grpc/types.ts

```diff
@@ -29,18 +29,6 @@ export const deserializeReq = <S extends ServiceType>(
   };
 };
 
-// If from dapp, send to tab
-// If internal message, send via chrome.runtime
-export const getTransport = (
-  sender: chrome.runtime.MessageSender,
-): ((res: unknown) => Promise<unknown>) => {
-  if (sender.tab?.id) {
-    return (res: unknown) => chrome.tabs.sendMessage(sender.tab!.id!, res); // Guaranteed given request is from dapp
-  } else {
-    return (res: unknown) => chrome.runtime.sendMessage(sender.id, res);
-  }
-};
-
```

Comment:
the existing router posted responses directly to the client, instead of returning. had to rip that out in order to comply with connection management.